### ISSUE TITLE
Missing a renderFooter prop

### ISSUE THREAD

**Issue by kfiroo** [FaridSafi/react-native-gifted-chat#170]
Hey @FaridSafi nice job! :)

I'm missing a way to render something in the footer of the `MessageContainer`.
My use case is the I want to render the avatars of all users that read all the messages (something like Facebook messenger does).
I used to be able to achieve this with the `renderFooter` on `GiftedMessenger`, any idea on how to do it with `GiftedChat`?

Thanks

**Comment by FaridSafi** [comment]
Hi @kfiroo, I am preparing a commit

**Comment by kfiroo** [comment]
@FaridSafi Thanks bro! Let me know if I can help! <3

**Comment by FaridSafi** [comment]
I pushed a `footer` prop in `master` branch and added example in [App.js](https://github.com/FaridSafi/react-native-gifted-chat/blob/master/example/App.js)

Can you please tell me if it's ok for you before publishing ?
```
npm install --save https://github.com/FaridSafi/react-native-gifted-chat
```

**Comment by kfiroo** [comment]
@FaridSafi works great.
I think we should call it `renderFooter` to make it more clear it expects a function and not a component.
Other then that it's perfect :)

**Comment by FaridSafi** [comment]
I published 0.0.3
- Adding a static bottom view as `renderFooter`

One of the next release will support `renderStatus`

**Comment by nandiniparimi1107** [comment]
@kfiroo could you please provide example code for renderFooter. This is my code:
<GiftedChat
        messages={this.giftedChatMessages}
        onSend={this.onSend}
        renderAvatar={null}
        renderChatFooter={this.renderFooter}
        onInputTextChanged={this.onInputTextChanged}
        user={{
          _id: this.userStore.loggedInUser.zynga_account_id,
        }}
        imageStyle={{}}
      />

 renderFooter() {
    if (this.opponentIsTyping) {
      return <Text>{this.opponentIsTyping}</Text>;
    }
    return null;
  }

Even if the this.opponentIsTyping is updated, renderFooter is not getting triggered.
Please help me with this.
